homeassistant/components/plex/__init__.py

```diff
@@ -34,7 +34,7 @@
     CONF_SERVER,
     CONF_SERVER_IDENTIFIER,
     DISPATCHERS,
-    DOMAIN as PLEX_DOMAIN,
+    DOMAIN,
```

Comment:
I think `DOMAIN as XXX` should only be used when importing from another component, not within the same component.